packages/tab-list/src/tab-list.ts

```diff
@@ -87,8 +99,6 @@ export class TabList extends Focusable {
     protected render(): TemplateResult {
         return html`
             <slot
-                @click=${this.onClick}
-                @keydown=${this.onKeyDown}
```

Comment:
This makes me sad, but somehow lit-element and ShadyDOM don't work well enough together to make these event handlers work. The ShadyDOM maintainers [fixed this on their end](https://github.com/webcomponents/shadydom/issues/252) but lit-element didn't work.

Reply:
I tried this, but it blew up the CSS pretty badly.